package/crds/elasticache.aws.upbound.io_usergroupassociations.yaml

```
@@ -0,0 +1,651 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.14.0
+  name: usergroupassociations.elasticache.aws.upbound.io
+spec:
+  group: elasticache.aws.upbound.io
+  names:
+    categories:
+    - crossplane
+    - managed
+    - aws
+    kind: UserGroupAssociation
+    listKind: UserGroupAssociationList
+    plural: usergroupassociations
+    singular: usergroupassociation
+  scope: Cluster
+  versions:
+  - additionalPrinterColumns:
+    - jsonPath: .status.conditions[?(@.type=='Synced')].status
+      name: SYNCED
+      type: string
+    - jsonPath: .status.conditions[?(@.type=='Ready')].status
+      name: READY
+      type: string
+    - jsonPath: .metadata.annotations.crossplane\.io/external-name
+      name: EXTERNAL-NAME
+      type: string
+    - jsonPath: .metadata.creationTimestamp
+      name: AGE
+      type: date
+    name: v1beta1
+    schema:
+      openAPIV3Schema:
+        description: UserGroupAssociation is the Schema for the UserGroupAssociations
```

@turkenf @mbbush - is that okay?
```suggestion
         description: |-
            UserGroupAssociation is the Schema for the UserGroupAssociations API. Associate an ElastiCache user and user group.
            NOTE:
            The API will detect changes in the UserGroups since UserGroupAssociations changes the user IDs associated with the user group.
            You can ignore these changes with the initProvider section as shown in the example.
```

---

You can't edit these files manually: they're generated. Instead you need to modify them in code while generation is happening. That's what https://github.com/crossplane-contrib/provider-upjet-aws/blob/45ec02187ac6371229b719fc3eb277914a666552/config/kafka/config.go#L234 does. As far as what the docs should say, that will depend on how we resolve the conflict between the two resources.